restapi/src/main/java/io/stargate/web/docsapi/service/DocumentService.java

```diff
@@ -476,15 +491,22 @@ private DocumentDB maybeCreateTableAndIndexes(
           docs.entrySet().stream()
               .map(
                   data -> {
-                    bindVariableList.addAll(
-                        shredJson(
-                                surfer,
-                                finalDb,
-                                Collections.emptyList(),
-                                data.getKey(),
-                                data.getValue(),
-                                false)
-                            .left);
+                    try {
+                      bindVariableList.addAll(
+                          shredJson(
+                                  surfer,
+                                  finalDb,
+                                  Collections.emptyList(),
+                                  data.getKey(),
+                                  data.getValue(),
+                                  false)
+                              .left);
+                    } catch (RuntimeException e) {
+                      throw new ErrorCodeRuntimeException(
+                          ErrorCode.DOCS_API_INVALID_JSON_VALUE,
```

Comment:
This is catching RuntimeException, do we need to check the type of the exception to see if it's json related or do we know for sure that's what gets thrown from here?

Reply:
I think I am going to change `shredJson` to just directly throw an `ErrorCodeRuntimeException`, but yes - because the function only handles JSON input (and JsonSurfer only throws plain RuntimeExceptions), I think it is safe to assume this